continuous-delivery/update-version.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
should this be python3?

https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take

Reply:
actually, the whole line seems unnecessary